Add playtest announcement (#46)
OpenRA/OpenRAWebsiteV3: news/_posts/2023-08-08-playtest-20230808.md

---
kind: article
title: "Playtest 20230808"
author: "PunkPun"
created_at: 2023-08-08 20:00 +0000
disqus_id: "playtest-20230801"
permalink: "/news/playtest-20230808/"
---

<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
<div style="margin: -10px 5px" markdown="1">

Another week, another playtest! Notable changes include:

* Added new Red Alert mission "Soviet13b"

> * Fixed dedicated servers not allowing maps with lua to be played

Reviewer: ```suggestion
* Fixed dedicated servers not allowing scripted maps to be played
```

> * Fixed the rare aircraft jitter 

Reviewer: Rare? Maybe this should also mention that this fixes visual jittering while steering. (I.e. be more formulated from a user POV that they understand what you mean here.)

Author: I'm really not sure how word it differently

> * Fixed map importer crashing when trying to convert maps with tile errors

Reviewer: ```suggestion
* Fixed map importer crashes when trying to convert maps with tile errors
```

(Or `Fixed the map importer crashing...`)

> * Fixed low power notification never playing

Reviewer: ```suggestion
* Fixed the low power notification never playing
```

For more information, see the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/b9b8aeb58667d08595d6471378fdf0d593b12de2).

And of course the Tiberian Dawn HD playtest can be [downloaded here](https://github.com/OpenRA/TiberianDawnHD/releases/tag/playtest-20230808). For more information see the [original playtest-20230801 announcement]({{ '/news/playtest-20230801/' | relative_url }}).

> In an effort of pushing out a swift release we highly encourage everybody to play some games and report any issues you may find in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or [GitHub](https://github.com/OpenRA/OpenRA/) issues. Good luck on the battlefield commanders!

Reviewer: ```suggestion
In an effort of pushing out a swift release we highly encourage everybody to play some games and report any issues you may find in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or [GitHub](https://github.com/OpenRA/OpenRA/). Good luck on the battlefield, commanders!
```
